scripts/variantstore/wdl/GvsCreateVAT.wdl

```diff
@@ -97,40 +99,42 @@ task MakeSubpopulationFiles {
         File inputFileofIndexFileNames
     }
     parameter_meta {
-        input_ancestry_file: {
-          localization_optional: true
+        input_ancestry_file:
+        {
+            localization_optional: true
         }
-        inputFileofFileNames: {
-          localization_optional: true
+        inputFileofFileNames:
```

Comment:
Not your doing, but don't like the mix of camel case and python style naming here.